analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AssertionArgsShouldBePassedInCorrectOrder.Xunit.cs

```diff
@@ -27,3 +27,30 @@ public void Foo()
         }
     }
 }
+
+// https://github.com/SonarSource/sonar-dotnet/issues/6547
+namespace Repro_6547
+{
+    class Program
+    {
+        enum Seasons
+        {
+            Spring,
+            Summer,
+            Autumn,
+            Winter
+        }
+
+        [Fact]
+        public void Repro_6547()
+        {
+            const string expected = "Spring";
```

Comment:
I suggest to use different names than `expected` and `actual`, because below you pass them both in the correct way and in the incorrect way to the `Same()` method, and that introduces confusion when reading the tests

Reply:
I think that naming the variable 'expected' allows us to immediately locate the expected value between the two and its position (first or second parameter). WDYT?

Reply:
You may use named arguments to signal the mixed-up parameter/arguments (Note: this might break the test, depending on how the rule is implemented). 
```cs
Assert.Same(expected: seasonString, actual: expected); // FN
```